deploy/openshift/templates/che-server-template.yaml

```diff
@@ -161,6 +161,12 @@ objects:
                 key: ca.crt
                 name: self-signed-certificate
                 optional: true
+          - name: CHE_CUSTOM_PUBLIC_CERT
+            valueFrom:
+              configMapKeyRef:
```

Comment:
`secretKeyRef` ?

Reply:
I don't see any reasons to use a secret. The certificate is public.

Reply:
In the code about `secretKeyRef:` is used

Reply:
fixed to configmap